test/test_playbooks/tasks/product.yml

```diff
@@ -4,6 +4,7 @@
     - organization_name: "Test Organization"
     - product_name: "Test Product"
     - product_description: "A happy little test product"
+    - product_content_credential: ""
```

Comment:
If you do not put that here, than we can test the behaviour without that option.
`omit` is like it was not specified at all.